docs/jwst/data_products/file_naming.rst

```diff
@@ -28,8 +30,16 @@ An example Stage 1 product FITS file name is::
 
 .. _src_file_names:
 
-Stage 3 Naming
-^^^^^^^^^^^^^^
+Association file names
+^^^^^^^^^^^^^^^^^^^^^^
+Associations capture the relationship between exposures that are to be combined by design to 
+form a single product. Association data products result from the combination of multiple exposures like dithers 
+or mosaics and are generated by the stage 3 of the calibration pipeline. 
+
+The format for the file name of an association has all alphabetic characters in lower case, underscores 
```

Comment:
"... of an association **product** has all alphabetic ..."

Reply:
changed it to "...Stage 3 association product..."